src/app/public/modules/radio/radio-group.component.spec.ts

```diff
@@ -0,0 +1,130 @@
+// #region imports
+import {
+  ComponentFixture,
+  fakeAsync,
+  TestBed,
+  tick,
+  async
+} from '@angular/core/testing';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  expect
+} from '@blackbaud/skyux-builder/runtime/testing/browser';
```

Comment:
This version of `expect` is deprecated; we're going to start telling people to use:

```
import {
  expect
} from '@skyux-sdk/testing';
```

Reply:
done